lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -0,0 +1,288 @@
+            },
+
+            successMessage: function() {
+                return this.successMessagePrefix + gettext('Successfully changed.');
+            },
+
+            message: function (message) {
+                return this.$('.account-settings-field-message').html(message);
+            },
+
+            showHelpMessage: function () {
+                this.message(this.helpMessage);
+            },
+
+            showInProgressMessage: function () {
+                this.message(this.inProgressMessage);
+            },
+
+            showSuccessMessage: function () {
+                this.message(this.successMessage());
+
+                if (this.options.refreshPageOnSave) {
+                    document.location.reload();
+                }
+
+                var view = this;
+
+                var context = Date.now()
+                this.lastSuccessMessageContext = context;
+
+                var successMessage = this.successMessage();
+
+                setTimeout(function () {
+                    if ((context === view.lastSuccessMessageContext) && (view.message().html() == successMessage)) {
+                        view.showHelpMessage();
+                    }
+                }, messageRevertDelay);
+            },
+
+            showErrorMessage: function (xhr) {
+                if (xhr.status === 400) {
+                    try {
+                        var errors = JSON.parse(xhr.responseText);
+                        var message = this.invalidInputMessagePrefix + errors['field_errors'][this.options.valueAttribute]['user_message'];
+                        this.message(message);
+                    } catch (error) {
+                        this.message(this.errorMessage);
+                    }
+                } else {
+                    this.message(this.errorMessage);
+                }
+            },
+        });
+
+        AccountSettingsFieldViews.ReadonlyFieldView = AccountSettingsFieldViews.FieldView.extend({
+
+            templateSelector: '#field_readonly-tpl',
+
+            initialize: function (options) {
+                this._super(options);
+                _.bindAll(this, 'render', 'updateValueInField');
+                this.listenTo(this.model, "change:" + this.options.valueAttribute, this.updateValueInField);
+            },
+
+            render: function () {
+                this.$el.html(this.template({
+                    title: this.options.title,
+                    value: this.modelValue(),
+                    message: this.helpMessage,
+                }));
+                return this;
+            },
+
+            updateValueInField: function () {
+                this.$('.account-settings-field-value').html(this.modelValue());
+            },
+        });
+
+        AccountSettingsFieldViews.TextFieldView = AccountSettingsFieldViews.FieldView.extend({
+
+            templateSelector: '#field_text-tpl',
+
+            events: {
+                'change input': 'saveValue',
```

Comment:
we usually do not put comma at the end of single element or the last element of javascript object.